public/img/logo.svg

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
In SVG files are a lot of crap. if you like you can remove the most of them. 
* All IDs
* All Meta descriptions
* the version
* <defs />